Rakefile

```diff
@@ -74,3 +72,107 @@ Rake::RDocTask.new do |rdoc|
   rdoc.rdoc_files.include('README*')
   rdoc.rdoc_files.include('lib/**/*.rb')
 end
+
+Rake::TestTask.new(:generate) do
+  $:.unshift File.join(File.dirname(__FILE__), "lib")
+  require "safemode"
+  default_methods = Safemode.class_variable_get("@@default_methods")
+  dangerous_methods = %w(
+  !~
+  =~
+  class
+  class_eval
+  clone
+  deep_clone
+  deep_dup
+  define_singleton_method
+  display
+  dup
+  enum_for
+  extend
+  gem
+  hash
+  __id__
+  inspect
```

Comment:
Inspecting is probably safe, however we do not want "attackers" to inspect things.